local/local.go

```diff
@@ -624,12 +629,26 @@ func untar(r io.Reader, dest string) error {
 			fh.Close()
 		case tar.TypeSymlink:
 			if err := os.Symlink(hdr.Linkname, path); err != nil {
```

Comment:
I don't understand why `os.Symlink` returns err when target doesn't exists.
When I tried to create new golang project it seems to not care about that. Also `ln` commands happily creates links with invalid destination.

Reply:
Ahh this might be happening because parent dir doesn't exist!

Reply:
Yep the tar I was receiving (from `podman`) is missing directories for some reason.

Reply:
The `untar` function does create parental dirs for regular files, but not for symlinks. This PR has to be reworked!